packages/analytics-browser/src/storage/local-storage.ts

```diff
@@ -1,8 +1,26 @@
 import { getGlobalScope } from '@amplitude/analytics-client-common';
 import { BrowserStorage } from './browser-storage';
+import { Logger } from '@amplitude/analytics-types';
 
+interface LocalStorageOptions {
+  logger?: Logger;
+}
 export class LocalStorage<T> extends BrowserStorage<T> {
-  constructor() {
+  logger?: Logger;
```

Comment:
i think we want the keys to be exactly what config uses?
```suggestion
  loggerProvider?: Logger;
```